build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    kotlin("jvm") version "1.7.10"
+    `java-gradle-plugin`
+    `maven-publish`
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    gradlePluginPortal()
+}
+
+dependencies {
+    implementation("com.google.protobuf:protobuf-java:2.6.1")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib")
+    testImplementation("org.jetbrains.kotlin:kotlin-stdlib")
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    implementation("org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.4.2")
+    testImplementation("junit:junit:4.12")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+publishing {
+    publications.create<MavenPublication>("maven") {
+        from(components["java"])
+    }
+}
```

Comment:
We should use the `-no-reflect` flag here to make sure we don't rely on `kotlin.reflect` anywhere. For now, I only see `import kotlin.reflect` in `unusedTestData` and `testData` (the latter should be investigated if it turns out these tests were not left in `testData` by mistake).